cmake/FindPyQt.py

```diff
@@ -27,19 +27,54 @@
 #
 # FindPyQt.py
 # Copyright (c) 2007, Simon Edwards <simon@simonzone.com>
+# Copyright (c) 2015, Alexei Ardyakov <ardjakov@rambler.ru>
 # Redistribution and use is allowed according to the terms of the BSD license.
 # For details see the accompanying COPYING-CMAKE-SCRIPTS file.
 
-import PyQt4.pyqtconfig
+import os
 
-pyqtcfg = PyQt4.pyqtconfig.Configuration()
-print("pyqt_version:%06.0x" % pyqtcfg.pyqt_version)
-print("pyqt_version_num:%d" % pyqtcfg.pyqt_version)
-print("pyqt_version_str:%s" % pyqtcfg.pyqt_version_str)
+def get_pyqt_sip_dir():
+    # Only works for a few popular locations of PyQt4 SIP directory.
+    import sipconfig
+    sipcfg = sipconfig.Configuration()
+    default_sip_dir = sipcfg.default_sip_dir
+    postfixes = ('Py2-PyQt4', 'PyQt4',)
+    for postfix in postfixes:
+        pyqt_sip_dir = os.path.join(default_sip_dir, postfix)
+        if os.path.exists(pyqt_sip_dir) and os.path.exists(
+                os.path.join(pyqt_sip_dir, 'QtCore')):
+            return pyqt_sip_dir
+    return default_sip_dir
+
+
+def get_pyqt_bin_dir():
+    import sipconfig
+    sipcfg = sipconfig.Configuration()
+    return sipcfg.default_bin_dir
```

Comment:
This does not work on Mac, especially for Homebrew where installs are symlinked to standard location. It returns `/usr/local/opt/python/Frameworks/Python.framework/Versions/2.7/bin` whereas PyQt utilities are installed elsewhere with the `--bindir=` options. Maybe do this, then test if a known utility exists, if not do a [generic platform-independent `which` search along PATH](http://stackoverflow.com/a/377028)?